### PR TITLE
gobrake-beego middleware and example application

### PR DESCRIPTION
**Pull request by chimanjain**
Added gobrake-beego middleware
Added sample application
Bumped supported beego version from v1 to v2
Removed older example application

### CODE REVIEW

File: examples/beego/main.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+	Hourly         []interface{} `json:"hourly,omitempty"`
+	Daily          []interface{} `json:"daily,omitempty"`
+}
+
+type Controller struct {
+	web.Controller
+}
+
+var ProjectId int64 = 363389
+var ProjectKey string = "baa755018d5e35e07897ee0087fcce9c"
```
Comment (andrew-werdna) [inline examples/beego/main.go:37]: These are a **real** `ProjectId` and `ProjectKey`, pointing to [this](https://getexceptional.airbrake.io/projects/363389) project. We should replace these with the usual example values. Then maybe re-base the new commit into the previous one so these values aren't visible.

**Review by andrew-werdna — CHANGES_REQUESTED** [review]
Nice cleanup of everything! I just had one comment that needs to be addressed before approving.

File: examples/beego/main.go (hunk 1 above)
Comment (chimanjain) [reply examples/beego/main.go:37]: Great catch, I will replace them. I will squash all the commits in `beego-integration` before merging into the `main` branch.

Comment (rbg) [reply examples/beego/main.go:37]: Regenerate the project key.

Comment (chimanjain) [reply examples/beego/main.go]: Yes

**Comment by chimanjain** [comment]
PTAL

**Review by andrew-werdna — APPROVED** [review]
LGTM!

**Pull request merged**